scripts/tests/perf/relayed-udp-server2client.sh

```diff
@@ -7,8 +7,7 @@ install_iptables_drop_rules
 
 docker compose exec --env RUST_LOG=info -it client /bin/sh -c 'iperf3 \
   --reverse \
-  --zerocopy \
   --udp \
-  --bandwidth 1G  \
+  --bandwidth 50M \
```

Comment:
It was failing occasionally at 100M :-/. I set it to 50M because that's what TCP seemed to be able to saturate.

When #3732 is merged, this could very well go up, but I think we are dropping too many packets otherwise.